crates/agglayer-aggregator-notifier/src/packer/mod.rs

```diff
@@ -113,78 +114,90 @@ where
                 proof.into(),
             );
 
+        let span = Span::current();
+        span.record("certificate_id", hash.clone());
+        span.record("network_id", *network_id);
+        if let Ok(data) = serde_json::to_string(&contract_call.function.inputs) {
+            span.record("call_data", data);
+        } else {
```

Comment:
I think `contract_call.function.inputs` seems to be the list of the input ABI variables but without their actual values, we may want what is under `contract_call.tx.data` instead if possible, they seem to provide one method `contract_call.call_data()`

Reply:
discussed offline: will get back to that if we fall back again to one InvalidProof